apps/backend/src/fund/entity/YieldBearingToken.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  getDiscriminatorModelForClass,
+  modelOptions,
+  prop,
+} from '@typegoose/typegoose';
+import { TokenEntity, TokenModel } from './Token';
+
+@modelOptions({
+  schemaOptions: {
+    toJSON: { virtuals: true },
+    toObject: { virtuals: true },
```

Comment:
My understanding is that virtuals allows the toJSON and toObject method to exist as computed props on the model, without persisting the values to the db?